src/main/java/FigureSupplier.java

```diff
@@ -0,0 +1,23 @@
+public class FigureSupplier {
+    private static final String[] nameFigure = new String[]{"circle", "square", "rectangle",
```

Comment:
в названиях констант все буквы должны быть большими, больше об этом можешь почитать здесь: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    private static final String[] FIGURES = new String[]{"circle", "square", "rectangle",
```